packages/grafana-ui/src/components/Table/Table.tsx

```diff
@@ -207,35 +218,51 @@ export const Table = memo((props: Props) => {
   useResetVariableListSizeCache(extendedState, listRef, data);
   useFixScrollbarContainer(variableSizeListScrollbarRef, tableDivRef);
 
-  const renderSubTable = useCallback(
+  const renderSubTables = useCallback(
     (rowIndex: number) => {
-      if (state.expanded[rowIndex]) {
-        const rowSubData = subData?.find((frame) => frame.meta?.custom?.parentRowIndex === rowIndex);
-        if (rowSubData) {
-          const noHeader = !!rowSubData.meta?.custom?.noHeader;
+      const subTables: ReactElement[] = [];
+
+      if (nestedFrames[rowIndex] && state.expanded[rowIndex]) {
+        let top = tableStyles.rowHeight; // initial height for row that expands above sub tables
+        nestedFrames[rowIndex].forEach((nf: DataFrame, nfIndex: number) => {
+          const noHeader = !!nf.meta?.custom?.noHeader;
```

Comment:
what is this meta.custom.noHeader property?

Reply:
It allows a user to hide a tables header, should they not want to show it in the nested fields for example. It's a remnant of the older sub tables work which I thought may be helpful.